docs/endpoints.rst

```diff
@@ -262,6 +262,23 @@ following:
    $ sudo systemctl restart my-globus-compute-endpoint.service
 
 
+AMQP Port
+---------
+
+.. TODO: fill in actual release number
+
+Endpoints receive tasks and communicate task results over AMQP. As of vX.X.X, that is
+done over port 443 by default, since firewall rules usually leave it open. In case 443
```

Comment:
Should we clarify that this is for newly created endpoints? ie. "...as of vX.X.X, that is done over port 443 by default for newly configured endpoints, since firewall..."